### PR TITLE
KEYCLOAK-12240 MigrationModelTest fails in pipeline

### PR DESCRIPTION
**Pull request by vramik**

### CODE REVIEW

**Comment by vramik** [comment]
https://keycloak-jenkins.com/job/universal-test-pipeline-server/1235/
https://keycloak-jenkins.com/job/test-keycloak-jpa/148/

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/model/MigrationModelTest.java [hunk 1]
```diff
@@ -49,6 +49,8 @@ public void test() {
 
             l = em.createQuery("select m from MigrationModelEntity m ORDER BY m.updatedTime DESC", MigrationModelEntity.class).getResultList();
             Assert.assertEquals(2, l.size());
+            System.out.println("%% " + l.get(0).getId() + "; " + l.get(0).getVersion() + "; " + l.get(0).getUpdateTime());
```
Comment (hmlnarik) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/model/MigrationModelTest.java]: Please use Logger instead.

**Review by hmlnarik — CHANGES_REQUESTED** [review]
Thanks for the fix! 
Please see inline for the biggest issue.
Nit: Perhaps it would be safer to use even a bigger time difference, if that helps.

**Review by hmlnarik — APPROVED** [review]
Thanks!

**Pull request merged**